From 80b1efa92486a87f9e82dbf665ef612291148de8 Mon Sep 17 00:00:00 2001
From: Adam Johnston <adam.johnston@arm.com>
Date: Tue, 14 Jun 2022 11:19:30 +0000
Subject: [PATCH] arm-bsp/trusted-firmware-a: N1SDP trusted boot

Increase max size of BL2 on N1SDP by 4KB to enable trusted boot
Decrease max size of BL1 on N1SDP by 8KB so BL1/BL2 fits above BL31 progbits

Signed-off-by: Adam Johnston <adam.johnston@arm.com>
Upstream-Status: Pending [Flagged to upstream]

---
 plat/arm/board/n1sdp/include/platform_def.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plat/arm/board/n1sdp/include/platform_def.h b/plat/arm/board/n1sdp/include/platform_def.h
index c9b81bafa..7468a31ed 100644
--- a/plat/arm/board/n1sdp/include/platform_def.h
+++ b/plat/arm/board/n1sdp/include/platform_def.h
@@ -91,7 +91,7 @@
  * PLAT_ARM_MAX_BL1_RW_SIZE is calculated using the current BL1 RW debug size
  * plus a little space for growth.
  */
-#define PLAT_ARM_MAX_BL1_RW_SIZE	0xE000
+#define PLAT_ARM_MAX_BL1_RW_SIZE	0xC000
 
 /*
  * PLAT_ARM_MAX_ROMLIB_RW_SIZE is define to use a full page
@@ -110,7 +110,7 @@
  * little space for growth.
  */
 #if TRUSTED_BOARD_BOOT
-# define PLAT_ARM_MAX_BL2_SIZE		0x20000
+# define PLAT_ARM_MAX_BL2_SIZE		0x21000
 #else
 # define PLAT_ARM_MAX_BL2_SIZE		0x14000
 #endif
-- 
2.35.1

